internal/utils/aggregator_test.go

```diff
@@ -112,43 +136,31 @@ func Test_filter_valid_k8s_object_types(t *testing.T) {
 		},
 	}
 	df = dataframe.LoadStructs(usage_data)
-	df = determine_k8s_object_type(df)
-	result = filter_valid_k8s_object_types(df)
+	result, _ = filter_valid_csv_records(df)
 	if result.Nrow() != 0 {
 		t.Error("Invalid k8s object type did not get dropped")
 	}
-}
 
-func Test_filter_valid_csv_records(t *testing.T) {
-	usage_data := []UsageData{
-		// k8s object with missing data
-		{
-			"2023-02-01 00:00:00 +0000 UTC", "2023-03-01 00:00:00 +0000 UTC", "2023-06-02 00:00:01 +0000 UTC", "2023-06-02 00:15:00 +0000 UTC",
-			"Yuptoo-service", "Yuptoo-app-standalone-1", "Yuptoo-app", "ReplicaSet", "testdeployment", "deployment", "Yuptoo-prod",
-			"quay.io/cloudservices/yuptoo", "ip-10-0-176-227.us-east-2.compute.internal", "i-0dfbb3fa4d0e8fc94",
-			"1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "",
-		},
+	// check if empty workload_type is dropped
```

Comment:
I don't think we'll need to check for the empty/`<none>` workloadType here, anymore.
This is because you have added the changes in `Test_Aggregate_data` that,

1. checks for empty, `<none>` workload_type
2. gracefully asserts that function does not panic

Reply:
hmm, we have tests for most of the filters? That does mean in future if someone removes these filters for dropping empty `workload_type` then the test should fail?
The Test_aggregate_data is using same `usage_data` however intention is different, and its about `gracefully asserts that function does not panic`